Command/ResetPasswordHandler.php

```diff
@@ -14,13 +15,22 @@ class ResetPasswordHandler
     private $userRepositoryCollection;
 
     /**
+     * @var EncoderFactoryInterface
```

Comment:
In de vorige commit ga je alles op met var op 1 lijn gaan zetten, hier dan mss ook doen?